cfme/cloud/instance/__init__.py

```diff
@@ -102,16 +70,19 @@ def in_cloud_instance(self):
 
 
 class InstanceAllView(CloudInstanceView):
+    """
+    The collection page for instances
+    """
     @property
     def is_displayed(self):
         return (
             self.in_cloud_instance and
             self.entities.title.text == 'All Instances' and
             self.sidebar.instances.is_opened)
 
-    toolbar = View.nested(InstanceToolbar)
+    toolbar = View.nested(VMToolbar)
     sidebar = View.nested(InstanceAccordion)
-    entities = View.nested(InstanceEntities)
+    entities = View.nested(VMEntities)
```

Comment:
Noted, I figured we would have some conflicting changes here.

I'll definitely wait for 4938 and inherit/extend/use the BaseEntities views/widgets.